scripts/tests/run_java_test.py

```diff
@@ -107,6 +108,15 @@ def main(app: str, app_args: str, tool_path: str, tool_cluster: str, tool_args:
         except Exception as e:
             logging.error(e)
             sys.exit(1)
+    elif tool_cluster == 'im':
```

Comment:
It looks all those four commands 'onnetwork-long-im-invoke/onnetwork-long-im-read/onnetwork-long-im-write/onnetwork-long-im-subscribe' are implemented within pairing cluster in the code

Reply:
update,d thanks